mlflow/tracking/fluent.py

```diff
@@ -323,6 +323,11 @@ def log_artifacts(local_dir, artifact_path=None):
     MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
 
 
+def _record_logged_model(mlflow_model):
```

Comment:
I wonder if we should remove this, it's a private method that's not used anywhere

Reply:
It's used in Model.log(...)

Reply:
Ah shoot searched for `mlflow._record_logged_model` instead of `mlflow.tracking.fluent._record_logged_model`. Makes sense, thanks!